SECURITY.md

```diff
@@ -0,0 +1,122 @@
+* **High-severity vulnerability**: The report identifies an exploitable security vulnerability that, if exploited, could result in any of the following:
+  * (Confidentiality) Unauthorized access to personal user data.
+  * (Integrity) Unauthorized edits to any user data.
+  * (Availability) Degraded system performance for the platform for users.
+
+Note that while the VRT contains multiple individuals, it’s ultimately expected that the VRT lead drives the effort. This should be in collaboration with VRT members, but in cases of urgent vulnerabilities the VRT lead can operate authoritatively to mitigate or remediate the issue (i.e. they do not need VRT consensus or review, but they should leverage VRT team members as a source for information and help).
+
+The VRT lead should notify the reporter of the classification and the reasoning behind the VRT’s decision within 7 days of the acknowledgement email. This notification should include links to any issues that were opened as a result of the report. For problems classified as bugs, feature requests, or low-severity vulnerabilities, issues should be opened and assigned to the relevant team.
+
+The rest of this document describes how we handle high-severity vulnerabilities.
+
+## Remediate
+
+### Create a Coordination Channel
+
+If the problem is confirmed as a high-severity vulnerability, the VRT will open a [GitHub security advisory](https://docs.github.com/en/code-security/repository-security-advisories/about-github-security-advisories-for-repositories) and, if both the VRT and reporter agree, add the reporter to the advisory so we can collaborate on it. We will coordinate work on the vulnerability on:
+
+* **The GitHub security advisory.** These advisories will let us collaborate in private, and they are appropriate in cases where disclosing the vulnerability prior to remediating it could put our users or developers at risk.
+* **(Optionally) An additional GitHub issue and pull request.** This will immediately disclose the vulnerability, and we will take this approach when immediate disclosure poses little risk to our users and developers. For example, when the vulnerability is already publicly known. Unlike security advisories, CI checks can run on these PRs.
+
+Note that we should create a GitHub security advisory even if we choose to collaborate on a fix using an open issue or PR because we’ll want to publish the advisory when disclosing the vulnerability.
+
+### Request a CVE
+
+A CVE should be requested for all security vulnerabilities. Since we create a GitHub Security Advisory, we can [get a CVE from GitHub](https://docs.github.com/en/code-security/repository-security-advisories/about-github-security-advisories-for-repositories#cve-identification-numbers). As a backup, we can [get a CVE from MITRE](https://cveform.mitre.org/).
```

Comment:
Should this be for all high-severity vulnerabilities? Earlier in the doc it's stated that this section be only followed for high-severity vulnerabilities, so if the intent is that both high & low severity have CVEs filed then some reworking is needed in the doc.